extensions/ql-vscode/test/vscode-tests/activated-extension/databases/db-panel.test.ts

```diff
@@ -15,11 +15,13 @@ import { createDbTreeViewItemSystemDefinedList } from "../../../../src/databases
 import { createRemoteSystemDefinedListDbItem } from "../../../factories/db-item-factories";
 import { DbConfigStore } from "../../../../src/databases/config/db-config-store";
 import { getActivatedExtension } from "../../global.helper";
+import { createVSCodeCommandManager } from "../../../../src/common/vscode/commands";
 
 jest.setTimeout(60_000);
 
 describe("Db panel UI commands", () => {
   let storagePath: string;
+  const commandManager = createVSCodeCommandManager();
```

Comment:
I believe we need to specify the type of the commands to ensure we actually have typing in tests as well:

```suggestion
  const commandManager = createVSCodeCommandManager<AllCommands>();
```